modules/transactions/src/main/java/org/apache/ignite/internal/tx/configuration/TransactionConfigurationSchema.java

```diff
@@ -31,13 +31,18 @@ public class TransactionConfigurationSchema {
     /** Default checking transaction interval. */
     public static final long DEFAULT_ABANDONED_CHECK_TS = 5_000;
 
-    /** Checking transaction interval. */
+    /** Checking transaction interval (milliseconds). */
```

Comment:
"Checking transaction interval" makes little sense to me, can you elaborate this please?